recipes/imagl/all/conanfile.py

```diff
@@ -32,7 +32,7 @@ def _build_subfolder(self):
     def _compilers_minimum_version(self):
         return {
             "gcc": "9",
-            "Visual Studio": "16.5",
+            "Visual Studio": "16",
```

Comment:
you can avoid to lose this information by using this pattern: https://github.com/conan-io/conan-center-index/blob/b5716c14ae9d40509374398a9cccf6e468a0f73f/recipes/sol2/3.x.x/conanfile.py#L33

Reply:
It *should* be OK to remove the ".5". The source code doesn't use new C++ features that are not in first Visual Studio 2019 release. Some newer features are used only if they are present in the compiler. I use some standard c++ definitions to test it.

Nevertheless, thank you for your help. I keep this tip in my mind.

(The ".5" comes from the fact that I was using Visual Studio 16.5 when I started to code imaGL)

Reply:
ok, then LGTM ;)